README.md

```diff
@@ -14,3 +14,11 @@ Coming soon:
 - tests for our code!
 - feedback box for more questions and other suggestions
 - moderated discussion page(s) 
+
+
+## Installation instructions (development)
+
+- edit django_project/sample_settings.py into settings.py
+- python manage.py migrate --run-syncdb
+- python manage.py loaddata data.json
+- python manage.py runserver
```

Comment:
then run these commands:
```
python manage.py migrate --run-syncdb
python manage.py loaddata data.json
python manage.py runserver
```

(the backticks make it a bit easier to see at a glance that you're running a command)